### ISSUE TITLE
NTR: Encapsulated budding yeasts on fluid or tissue specimen (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8106]
New term request Encapsulated budding yeasts on fluid or tissue specimen (nadea3_190725193555): Cryptococcus, cryptococcal  UMLS: 1545145

**Comment by pnrobinson** [comment]
@MickeySegal
what tissue is meant here (if there is a specific tissue)? Could you provide more context?

**Comment by MickeySegal** [comment]
https://www.merckmanuals.com/professional/infectious-diseases/fungi/cryptococcosis
"Diagnosis is strongly suggested if experienced observers identify encapsulated budding yeasts in smears of body fluids, secretions, exudates, or other specimens."

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000964
- label: Positive tissue encapsulated budding yeast test
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Identification of encapsulated budding yeasts in smears of body fluids, secretions, exudates, or other specimens.
- comment: In fixed-tissue specimens, encapsulated yeasts may also be identified and confirmed as cryptococci by positive mucicarmine or Masson-Fontana staining. This finding may suggest a diagnosis of cryptococcosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8106
- synonyms:
